test/e2e/route_advertisements.go

```diff
@@ -963,14 +965,20 @@ var _ = ginkgo.DescribeTableSubtree("BGP: isolation between advertised networks"
 					errBool := false
 					out := ""
 					if cudnATemplate.Spec.Network.Topology == udnv1.NetworkTopologyLayer2 {
-						// FIXME: fix assymmetry in L2 UDNs
-						// bad behaviour: packet is coming from other node -> entering eth0 -> bretho and here kernel drops the packet since
-						// rp_filter is set to 1 in breth0 and there is an iprule that sends the packet to mpX interface so kernel sees the packet
-						// having return path different from the incoming interface.
-						// The SNAT to nodeIP should fix this.
-						// this causes curl timeout with code 28
-						errBool = true
-						out = curlConnectionTimeoutCode
+						// FIXME: this should be removed once we add the SNAT for pod->node traffic
+						// We now permit asymmetric traffic on LGW. This prevents the issue from occurring with IPv6.
+						// However, for IPv4 LGW rp_filter is still blocking the replies.
+						// The situation is different on SGW as we don't allow asymmetric traffic at all, which is why IPv6 traffic fails there too.
+						if ipFamilyIndex == ipFamilyV4 || !isLocalGWModeEnabled() {
+							// FIXME: fix assymmetry in L2 UDNs
```

Comment:
I think we are going to have conflicts for sure hehe.. I hope LGW with L2 doesn't give me any surprises for https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5140

https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5140/commits/9e2923971a6ab970d31d88aba1dfa057cc377092#diff-00535718a80e537658f346b0cbae8ced845dcc0cccc9cf79bd9fe610d09a6497R966

it all gets fixed here.. so between these two PRs I think we need to decide which PR should land first.